org.roojs.roobuilder.json

```diff
@@ -0,0 +1,82 @@
+{
+    "app-id": "org.roojs.roobuilder",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+    	"org.freedesktop.Sdk.Extension.vala"
```

Comment:
vala is included in the GNOME runtime

Reply:
build seems to fail if this is not added  - also not sure if vala-language-server is in them main runtime

Running custom install script '/usr/bin/gtk4-update-icon-cache -q -t -f /app/share/icons/hicolor'
cp: cannot stat '/usr/lib/sdk/vala/bin/.': No such file or directory

Reply:
It is in `/usr/bin`.

```sh
→ flatpak run --command=ls org.gnome.Sdk//45 -lA /usr/bin/vala
lrwxrwxrwx. 1 nfsnobody nfsnobody 9 May 10 20:00 /usr/bin/vala -> vala-0.56
~ 
→ flatpak run --command=ls org.gnome.Sdk//46 -lA /usr/bin/vala
lrwxrwxrwx. 1 nfsnobody nfsnobody 9 May 10 20:00 /usr/bin/vala -> vala-0.56
```